Documentation/README.md

```diff
@@ -20,6 +20,10 @@ This directory contains documentation for various topics related to SwiftGen, su
 * Learn how to [create your own custom templates](Articles/Creating-custom-templates.md) to customize the code SwiftGen generates
 * Iterate quickly when creating custom templates by [watching a file or folder for changes](Articles/Watch-a-folder-for-changes.md)
 
+### Customise loading of resources
```

Comment:
Just saw that 2 lines above we use the US spelling (`customize`) while here we use the UK one (with `s`). I don't mind which one we use tbh, but we might at least want to uniformize (uniformise? 😅) our spelling across the docs tho.